config/prow/config.yaml

```diff
@@ -226,3 +226,40 @@ presets:
   env:
     - name: GOPPC64
       value: power9
+# knative powervs preset
+- labels:
+    preset-knative-powervs: "true"
+  env:
+    - name: TF_VAR_powervs_api_key
+      valueFrom:
+        secretKeyRef:
+          name: knative-apikey
+          key: apikey
+    - name: KO_FLAGS
+      value: --platform=linux/ppc64le
+    - name: PLATFORM
+      value: linux/ppc64le
+    - name: INGRESS_CLASS
+      value: contour.ingress.networking.knative.dev
+    - name: KO_DOCKER_REPO
+      value: icr.io/upstream-k8s-registry/knative
```

Comment:
I still feel that you folks are just wasting resources by pushing these images to ICR if not been used by any endusers, see if you can use any other repos like quay/ghcr